src/Aquifer.Public.API/Endpoints/Bibles/List/Request.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public class Request
+{
+    /// <summary>
+    /// Optional language to search on by internal Aquifer language ID. If no language is specified then Bibles for all languages will be returned. Only one of language ID or code may be provided.
+    /// </summary>
+    public int? LanguageId { get; init; }
+
+    /// <summary>
+    /// Optional language to search on by ISO 639-3 code that is 3 characters in length (e.g. "eng"). If no language value is specified then Bibles for all languages will be returned. Only one of language ID or code may be provided.
+    /// </summary>
+    public string? LanguageCode { get; init; }
```

Comment:
I decided to support both `LanguageId` and `LanguageCode` to match the [Search Resources](https://github.com/BiblioNexusStudio/aquifer-server/blob/0f9cb13ccf06e6ba1a82e5131a3acd0fa0cfef04/src/Aquifer.Public.API/Endpoints/Resources/Search/GetResources/Request.cs#L44-L53) endpoint.